content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,532 @@
+---
+title: Access your log data programmatically using Logs Search API.
+kind: guide
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Search Syntax"
+- link: "/logs/search_syntax/"
+  tag: "Documentation"
+  text: "Learn more about syntax for Logs Search API"
+---
+
+
+## Overview
+
+Use the [Logs Search API][1] to programmatically access your log data and execute queries.
+
+The following examples are covered in this guide:
+
+* [Basic search](#basic-search)
+* [Sort by facet or timestamp](#sort-by-facet-or-timestamp)
+* [Limit the number of results retrieved](#limit-the-number-of-results-retrieved)
+* [Pagination](#pagination)
+
+## Prerequisites
+
+Since this guide describes usage of the API, you will need an API key and an application key from an admin user. These are available in your [Datadog account API key page][2].
+
+Throughout this article, you will need to replace all occurrences of `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key.
+
+This guide also assumes that you have a terminal with `CURL`. 
+
+**Note:** If you are in the Datadog EU site, use https://api.datadoghq.eu/api/ as the endpoint.
```

Comment:
We should have something specific to change the values based on the navigation settings which would prevent having to add notes